Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,89 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
+    let fileManager = FileManager.default
```

Comment:
Do we need this in this class, or can we move it to a local variable in the [`allFiles` function](https://github.com/xing/calabash-launcher/pull/148/files#diff-a3c8f30876102abb19640ca6e2426898R77)?